jf_agent/config_file_reader.py

```diff
@@ -371,6 +388,24 @@ def get_ingest_config(
     if config.jira_url and (
         (creds.jira_username and creds.jira_password) or creds.jira_bearer_token
     ):
+        issue_metadata: List[IssueMetadata] = IssueMetadata.from_json(
```

Comment:
Q: Is this to fix a change in jf_ingest?

Reply:
Also did we do a quick Jira test to validate this?

Reply:
This is logic we were already doing, extracting the `issue_metadata`

Directly below it, though, we're building a new dictionary of "project IDs to pull froms", which is something the new jira sync uses (Kirk's speedup). We need to supply JF Ingest with a pull_from date for each project in order to get all the updated issues for that project

The new sync is currently not used in Agent, so this is pretty low impact. I did, however, do a Jira test with orthogonal networks just to make sure and it behaved as expected